srcpkgs/apparmor/files/profiles/usr.bin.pulseaudio

```diff
@@ -67,12 +71,17 @@ profile pulseaudio /usr/bin/pulseaudio {
   /sys/devices/virtual/sound/**/uevent r,
 
   /usr/share/alsa/** r,
-  /usr/share/applications/ r,
-  /usr/share/applications/* r,
   /usr/share/pulseaudio/** r,
   /usr/lib/pulse-[1-9]*.[0-9]/modules/*.so mr,
   /usr/libexec/pulse/gsettings-helper Cx,
 
+  /usr/{,local/}share/applications/ r,
+  /usr/{,local/}share/applications/* r,
+  @{HOME}/.local/share/{,flatpak/exports/share/}applications/ r,
+  @{HOME}/.local/share/{,flatpak/exports/share/}applications/* r,
```

Comment:
change to `owner @{HOME}...` so that it only gives permissions for the user's own home directory.